packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css

```diff
@@ -25,6 +25,30 @@
 
 .navigation-header {
```

Comment:
It still looks slightly more centered in our designs 
| Designs |  Implementation | 
|---|---|
|  <img width="365" alt="Screenshot 2023-09-15 at 11 37 31" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/20dc8475-55f7-4bd8-9ada-22b6c8074092"> |  <img width="302" alt="Screenshot 2023-09-15 at 11 37 46" src="https://github.com/commercetools/merchant-center-application-kit/assets/49066275/f57dabcf-9998-4b90-81eb-6de989fbb39d"> |

Reply:
Oh yes I noticed the misalignment,
Thinking about it now, the content should be centered and not aligned with the contents on the side nav. 
Thank you

Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3216/commits/40c59028af074317acb832b4b353589510cecfe6